anyway/widgets/suburban_widgets/accident_count_by_car_type_widget.py

```diff
@@ -46,57 +52,71 @@ def get_stats_accidents_by_car_type_with_national_data(
         data_by_segment = AccidentCountByCarTypeWidget.percentage_accidents_by_car_type(
             involved_by_vehicle_type_data
         )
-        national_data = AccidentCountByCarTypeWidget.percentage_accidents_by_car_type_national_data_cache(
-            start_time, end_time
+        national_data = (
+            AccidentCountByCarTypeWidget.percentage_accidents_by_car_type_national_data_cache(
+                start_time, end_time
+            )
         )
 
-        for k, v in national_data.items():  # pylint: disable=W0612
+        for car_type in data_by_segment:
             out.append(
                 {
-                    "car_type": k,
-                    "percentage_segment": data_by_segment[k],
-                    "percentage_country": national_data[k],
+                    "label_key": car_type,
+                    "series": [
+                        {"label_key": "percentage_segment", "value": data_by_segment[car_type]},
+                        {"label_key": "percentage_country", "value": national_data[car_type]},
+                    ],
                 }
             )
         return out
 
     @staticmethod
     def percentage_accidents_by_car_type(involved_by_vehicle_type_data):
-        driver_types = defaultdict(float)
+        vehicle_type_dict = defaultdict(float)

```

Comment:
why float?

Reply:
That's the original interface, there was no request to change it